pkgs/applications/misc/moolticute/default.nix

```diff
@@ -1,36 +1,44 @@
-{ stdenv, fetchurl
-, libusb1, pkgconfig, qmake, qtbase, qttools, qtwebsockets
+{ lib, mkDerivation, fetchFromGitHub
+, libusb1
+, pkg-config
+, qmake
+, qtbase
+, qttools
+, qtwebsockets
 }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
```

Comment:
have I missed something, is this new official mkDerivation style?

Reply:
for qt packages, yes. This mkDerivation is actually different than stdenv.mkDerivation.

Reply:
right! thanks. I wish it was called `mkQtDerivation` https://github.com/NixOS/nixpkgs/issues/65399#issuecomment-526649894

Reply:
as do i, it causes a lot of confusion. Or `buildQtApplication`